skimage/restoration/deconvolution.py

```diff
@@ -365,13 +365,22 @@ def richardson_lucy(image, psf, iterations=50, clip=True):
     ----------
     .. [1] http://en.wikipedia.org/wiki/Richardson%E2%80%93Lucy_deconvolution
     """
+    direct_time = lambda n, m, k, l: k*l * n*m
+    def fft_time(m, n, k, l):
+        return m*np.log(m) + n*np.log(n) + k*np.log(k) + l*np.log(l)
+
+    time_ratio = 71.468 * fft_time(*image.shape, *psf.shape)
```

Comment:
@jni thanks for the improvement. I think I had seen this syntax (casting tuples to list to add them and then to tuple again) somewhere else in skimage's code and since then I replicated this pattern. I'll have a quick look to see if we can discard other clumsy castings of the same type.